controllers/velero.go

```diff
@@ -531,3 +530,31 @@ func (r *VeleroReconciler) getVeleroResourceReqs(velero *oadpv1alpha1.Velero) co
 
 	return ResourcesReqs
 }
+
+// For later: Move this code into validator.go when more need for validation arises
+// TODO: if multiple default plugins exist, ensure we validate all of them.
+// Right now its sequential validation
+func (r *VeleroReconciler) ValidateVeleroDeployment(log logr.Logger) (bool, error) {
```

Comment:
Consider renaming this to `ValidateVeleroPiugins` since that's what it's doing today. We can then add a generic `ValidateVeleroConfig` or something down the line thats calls this when we have more validations to add.